test/atom_finder/classifier_test.clj

```diff
@@ -172,3 +172,40 @@
         (testing (str "Is reversed-subscript-atom? - " code " - " sc?)
           (is (= (reversed-subscript-atom? (parse-expr code)) sc?))))
       )))
+
+(deftest test-omitted-curly-braces-atom?
+  (testing "omitted-curly-braces-atom? finds all atoms in snippet study code"
+
+    (let [lines  (->> (resource-path "omitted-curly-braces.c")
+                      tu
+                      (filter-tree omitted-curly-braces-atom?)
+                      (map loc)
+                      (map :line))]
+
+      (is (= lines [3 4 7 8 26 27 28 29 41 43 44 54 66 79 95 101 102 104 105 108 109])))))
+
+
+(deftest comma-operator-test
+  (testing "small statements" 
+    (is (comma-operator-atom? (parse-expr "1,2")))
+    (is (false? (comma-operator-atom? (parse-expr "int a,b;"))))
+    (is (comma-operator-atom? (parse-expr "num1,num2")))
+    (let [lines  (->> (resource-path "comma-operator.c")
+                      tu
+                      (filter-tree comma-operator-atom?)
+                      (map loc)
+                      (map :line))]
+
+      (is (= lines [4 11 11 11])))))
+
+(deftest test-assignment-as-value-atom?
+  (testing "assignment-as-value-atom? finds all atoms in snippet study code"
+
+    (let [lines  (->> (resource-path "assignment-as-value.c")
+                      tu
+                      (filter-tree assignment-as-value-atom?)
+                      (map loc)
+                      (map :line))]
+
+      (is (= lines [8 10 12 12 14 16 20 22 30 32 36 38 42 48 50 52 52 56 58 60 62])))))
```

Comment:
Hmm, not sure about the two atoms on same line, you might actually have to update the macro to get that to work since none of my previous atoms had that behavior.

Sometimes with cider you need to go to the file where the function/macro is defined in (I think it's test_util.clj) and eval (run) the definition. If that doesn't work, also try eval'ing the `(ns)` call at the top of the file in classifiers.clj and at the top of the file you're working in.